controllers/argocd/role.go

```diff
@@ -104,6 +104,11 @@ func (r *ReconcileArgoCD) reconcileRole(name string, policyRules []v1.PolicyRule
 
 	// create policy rules for each namespace
 	for _, namespace := range namespaces.Items {
+		var useCustomRole bool
+		if customRole := getCustomRoleName(name); customRole != "" {
+			useCustomRole = true
+		}
```

Comment:
Any special reason for having the `useCustomRole` boolean, and not just using check for `customRole != ""` below?

Reply:
It's not required. I will get rid of it